src/main/java/com/strandls/document/pojo/DocumentListParams.java

```diff
@@ -0,0 +1,29 @@
+package com.strandls.document.pojo;
+
+
+/**
+ * 
+ * @author vishnu
+ *
+ */
+
+
+public class DocumentListParams {
+	private String location ;
```

Comment:
use of this class ??? only single variable is present?

Reply:
//